src/api/handlers/event.py

```diff
@@ -253,6 +253,7 @@ def list(self, request):
     self.validator.expect("community_id", is_required=False)
     self.validator.expect("subdomain", is_required=False)
     self.validator.expect("user_id", is_required=False)
+    self.validator.expect("limit", is_required=False)
```

Comment:
maybe consider extracting the `limit` and `cursor` or `meta` in the context middle layer.  And set it there.  that means by default you make the change in one place and can get it anywhere there is a context.